### ISSUE TITLE
Seed values

### ISSUE THREAD

**Issue by MMesgar** [GT-SALT/IDBR#1]
Hi, 
Thanks for sharing the code of the models you used in the paper. I'm wondering what seed values did you use to run models three times? I tried with different seed values (seed=0,1,2) but the average of the scores that I get is different from what you have in the paper. 
For example for task order 1 (ag-yelp-yahoo) the FineTune model in Table 3 achieves 25.79% acc. I get  28.79% acc. 

Looking forward to hearing form you,

**Comment by StevenyzZhang** [comment]
Thank you for your interest in this work!


To clarify, I think we used seed=0,1,2 in our experiments.


For the reason why you and we got different numbers: 

You can ONLY get the same numbers with us while you are using the same software setting (e.g., PyTorch version, CUDA version, and every related library version) AND the same hardware setting (e.g., The graphics card type and CPU type) as we did, which is almost impossible in practice. 

The performance gap could be relatively big in those situations where variance is big. The FineTune model, as you mentioned, is possible to have this issue, since it sequentially learns several tasks without replay or regularization, thus tend to have big performance variance.


Also, considering the situation above, I suggest you verify the validity of the conclusion (IDBR is better than regularization, regularization is better than replay, replay is better than finetune), INSTEAD OF trying to reproduce exactly the same number.
